plugin/cmd/kube-scheduler/app/configurator.go

```diff
@@ -139,17 +141,13 @@ func (sc schedulerConfigurator) getSchedulerPolicyConfig() (*schedulerapi.Policy
 			return nil, fmt.Errorf("Error getting scheduler policy ConfigMap: %v.", err)
 		}
 		if policyConfigMap != nil {
-			// We expect the first element in the Data member of the ConfigMap to
-			// contain the policy config.
-			if len(policyConfigMap.Data) != 1 {
-				return nil, fmt.Errorf("ConfigMap %v has %v entries in its 'Data'. It must have only one.", sc.policyConfigMap, len(policyConfigMap.Data))
+			val, ok := policyConfigMap.Data[SchedulerPolicyConfigMapKey]
```

Comment:
replace `ok` with `policyConfigMapFound`; and remove `policyConfigMapFound = true`.